velox/functions/lib/window/NthValue.cpp

```diff
@@ -57,11 +58,15 @@ class NthValueFunction : public exec::WindowFunction {
                 ->valueAt(0);
         VELOX_USER_CHECK_GE(
             constantOffset_.value(), 1, "Offset must be at least 1");
-        return;
+      } else {
```

Comment:
Does 'return' on L53 needd to be removed as well?

Reply:
If constant offset is null, then all results are null irrespective of ignoreNulls. So the nullsVector is not needed.